From: Mohamad Al-Jaf <mohamadaljaf@gmail.com>
Subject: [PATCH v11 2/8] wdscore: Add wdscore_internal.h file.
Message-Id: <20220412030059.405645-1-mohamadaljaf@gmail.com>
Date: Mon, 11 Apr 2022 23:00:59 -0400

Signed-off-by: Mohamad Al-Jaf <mohamadaljaf@gmail.com>
---
v11: - Don't move the assembly code to the header.

The way I was testing this gave me a false ok. I didn't
think it through completely. Doesn't help that I'm
sleep-deprived and hadn't eaten anything all day.

At this point I'm pretty sure my patches are automatically
getting sent straight to /dev/null :(
---
 dlls/wdscore/wdscore_internal.h | 24 ++++++++++++++++++++++++
 1 file changed, 24 insertions(+)
 create mode 100644 dlls/wdscore/wdscore_internal.h

diff --git a/dlls/wdscore/wdscore_internal.h b/dlls/wdscore/wdscore_internal.h
new file mode 100644
index 00000000000..11111111111
--- /dev/null
+++ b/dlls/wdscore/wdscore_internal.h
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2022 Mohamad Al-Jaf
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+#ifndef __WDSCORE_INTERNAL_H__
+#define __WDSCORE_INTERNAL_H__
+
+LPVOID WINAPI CurrentIP(void);
+
+#endif /* __WDSCORE_INTERNAL_H__ */

-- 
0.0.0

